### PR TITLE
Water spring propagation fix

### PR DESCRIPTION
**Pull request by ghost**
Fixes [MC-916](https://bugs.mojang.com/browse/MC-916).

More details [here](https://www.minecraftforum.net/forums/mapping-and-modding-java-edition/minecraft-mods/1283147-water-spring-propagation-fix).

### CODE REVIEW

File: scripts/importmcdev.sh [hunk 1]
```diff
@@ -41,6 +41,7 @@ import NextTickListEntry
 import ThreadServerApplication
 import BlockChest
 import BlockGrass
+import BlockFlowing
```
Comment (js6pak) [inline scripts/importmcdev.sh:44]: Shouldn't this be removed?

Comment (astei) [reply scripts/importmcdev.sh:44]: Due to the fashion in which I had to lobotomize the Paper build system, this line is required.

Comment (js6pak) [reply scripts/importmcdev.sh:44]: So maybe 5th step from there https://github.com/canyonmodded/canyon/blob/master/CONTRIBUTING.md#help-i-cant-find-the-file-im-looking-for should be removed? (or i'm misunderstanding it?)

Comment (astei) [reply scripts/importmcdev.sh:44]: Fifth step is to "edit away" - isn't that the point of doing a patch? That's literally the last item on the list.

Comment (js6pak) [reply scripts/importmcdev.sh:44]: Ohh, nevermind im dumb

**Comment by astei** [comment]
Tested this out. Good fix!

**Pull request merged**